src/frontend/components/LiveVideoPanel/index.spec.tsx

```diff
@@ -0,0 +1,143 @@
+    });
+
+    const { container } = render(
+      wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />),
+    );
+
+    expect(mockSetPanelVisibility).toBeCalled();
+    expect(mockSetPanelVisibility).toBeCalledTimes(1);
+    expect(mockSetPanelVisibility).toBeCalledWith(false);
+
+    expect(container.hasChildNodes()).toBe(false);
+  });
+
+  it('renders the content with selection', () => {
+    useLivePanelState.setState({
+      currentItem: LivePanelItem.APPLICATION,
+      availableItems: [
+        LivePanelItem.APPLICATION,
+        LivePanelItem.CHAT,
+        LivePanelItem.JOIN_DISCUSSION,
+      ],
+    });
+
+    render(wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />));
+
+    screen.getByRole('tablist');
+    screen.getByRole('tab', { name: 'application' });
+    screen.getByRole('tab', { name: 'chat' });
+    screen.getByRole('tab', { name: 'viewers' });
+
+    screen.getByText('application content');
+  });
+
+  it('does not render tabs with only one item available', () => {
+    useLivePanelState.setState({
+      currentItem: LivePanelItem.APPLICATION,
+      availableItems: [LivePanelItem.APPLICATION],
+    });
+
+    render(wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />));
+
+    expect(screen.queryByRole('tablist')).toBeNull();
+    expect(screen.queryByRole('tab', { name: 'application' })).toBeNull();
+    expect(screen.queryByRole('tab', { name: 'chat' })).toBeNull();
+    expect(screen.queryByRole('tab', { name: 'viewers' })).toBeNull();
+
+    screen.getByText('application content');
+  });
+
+  it('does not render tabs on small screens', () => {
+    useLivePanelState.setState({
+      currentItem: LivePanelItem.APPLICATION,
+      availableItems: [
+        LivePanelItem.APPLICATION,
+        LivePanelItem.CHAT,
+        LivePanelItem.JOIN_DISCUSSION,
+      ],
+    });
+
+    const tree = renderer
+      .create(wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />))
+      .toJSON();
+
+    expect(
+      (tree as renderer.ReactTestRendererJSON).children![0],
+    ).toHaveStyleRule('display', 'none', {
+      media: '(max-width: 768px)',
+    });
+  });
+
+  it('renders with appropriate style on large screen', async () => {
+    useLivePanelState.setState({
+      currentItem: LivePanelItem.APPLICATION,
+      availableItems: [
+        LivePanelItem.APPLICATION,
+        LivePanelItem.CHAT,
+        LivePanelItem.JOIN_DISCUSSION,
+      ],
+    });
+
+    await renderImageSnapshot(
+      wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />),
+    );
+  });
+
+  it('renders with appropriate style on small screen', async () => {
+    useLivePanelState.setState({
+      currentItem: LivePanelItem.APPLICATION,
+      availableItems: [
+        LivePanelItem.APPLICATION,
+        LivePanelItem.CHAT,
+        LivePanelItem.JOIN_DISCUSSION,
+      ],
+    });
+
+    await renderImageSnapshot(
+      wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />),
+      300,
+      300,
+    );
```

Comment:
Can we have a test with no available items

Reply:
good catch, I'm questioning myself because the store should (or is expected as an invariant) to set the currentItem to undefined if there is no available items.
Then there is already a test checking the panel closes and do not render if the current Item is undefined